tox.ini

```diff
@@ -65,9 +65,8 @@ deps =
     twine: twine
 
     lint: pyflakes
-    lint: twistedchecker>=0.7.4
-    lint: diff-cover==0.9.12
-    lint: pycodestyle
+
+    black: black>=19.3b0
```

Comment:
Further down in the `deps` section, you have: `black==20.8b1`.
Might be good to keep this consistent

Reply:
Thanks, I'll clean this up.